<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionDialog</class>
 <widget class="QDialog" name="QuestionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="topWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>670</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QWidget" name="icon" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbTitle">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>5</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>360反馈建议</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbClose">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>7</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pbMin">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>7</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pbLuntan">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="mainWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>670</width>
     <height>430</height>
    </rect>
   </property>
   <widget class="QWidget" name="next" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>670</width>
      <height>460</height>
     </rect>
    </property>
    <widget class="QWidget" name="nextIcon" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="nextTitle">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>您有什么建议想对我们说</string>
     </property>
    </widget>
    <widget class="QLabel" name="nextPhone">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>你的手机型号:</string>
     </property>
    </widget>
    <widget class="QLabel" name="nextQQ">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>你的联系方式:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>90</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="nextTextEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>631</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="nextScreenshot">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>280</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="nextPhoto">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>280</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="nextAnnex">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>280</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="nextOther">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>621</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>请尽量留下您的电话,QQ等联系方式,如出现问题时的截图,以方便我们查找问题的原因.</string>
     </property>
    </widget>
    <widget class="QToolButton" name="nextPost">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>371</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nextBack">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>380</y>
       <width>91</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;&lt;  返回</string>
     </property>
    </widget>
    <widget class="QComboBox" name="nextComboBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>104</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="home" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>670</width>
      <height>430</height>
     </rect>
    </property>
    <widget class="QWidget" name="question" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>271</width>
       <height>431</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>0</y>
       <width>401</width>
       <height>421</height>
      </rect>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>8</y>
        <width>401</width>
        <height>421</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout"/>
     </widget>
    </widget>
   </widget>
   <zorder>home</zorder>
   <zorder>next</zorder>
  </widget>
  <zorder>mainWidget</zorder>
  <zorder>topWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
